src/Sylius/Bundle/ShopBundle/Resources/views/Common/Order/Label/PaymentState/orderPaymentState.html.twig

```diff
@@ -0,0 +1,16 @@
+{%
+    set viewOptions = {
+        awaiting_payment: {icon: "clock icon", color: "grey"},
+        partially_authorized: {icon: "check icon", color: "yellow"},
+        authorized: {icon: "check icon", color: "orange"},
+        partially_paid: {icon: "adjust icon", color: "olive"},
+        cancelled: {icon: "ban icon", color: "red"},
+        paid: {icon: "check icon", color: "green"},
+        partially_refunded: {icon: "reply icon", color: "violet"},
+        refunded: {icon: "reply all icon", color: "purple"},
+    }
+%}
+    <div class="ui top attached label {{ 'viewOptions.' ~ state ~ '.color' }}" style="margin-left: 1rem; width: calc(100% - 2rem); margin-top: 1rem;">
+    <i class={{ 'viewOptions.' ~ state ~ '.icon' }}></i>Status: <span id="order-payment-status">{{ value|trans }}</span>
```

Comment:
I would indent this line

Reply:
And let's keep the consistency between this and the next file, so get the color and icon data by `viewOptions[state][icon/color]`